recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+        - test -f ${PREFIX}/{{ install_base }}/Shared/InlineNullValues.h  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/Logger/Logger.h  # [unix]
+        - if not exist %PREFIX%\Shared\funcannotations.h exit 1  # [win]
+        - if not exist %PREFIX%\Shared\InlineNullValues.h exit 1  # [win]
+        - if not exist %PREFIX%\Logger\Logger.h exit 1  # [win]
+        # QE
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/heavydbTypes.h  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/RuntimeFunctions.bc  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/GeosRuntime.bc  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/ExtensionFunctions.ast  # [unix]
+        - if not exist %PREFIX%\QueryEngine\heavydbTypes.h exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\RuntimeFunctions.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\GeosRuntime.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\ExtensionFunctions.ast exit 1  # [win]
+        # jar
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavyai-utility-{{ base_version }}.jar  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavyai-jdbc-{{ base_version }}.jar  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/calcite-1.0-SNAPSHOT-jar-with-dependencies.jar  # [unix]
+        - if not exist %PREFIX%\bin\heavyai-utility-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\heavyai-jdbc-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\calcite-1.0-SNAPSHOT-jar-with-dependencies.jar exit 1  # [win]
+        # Unspecified
+        # startheavy and insert_sample_data are bash scripts
+        - test -f ${PREFIX}/{{ install_base }}/bin/startheavy  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavydb_insert_sample_data  # [unix]
+        #
+        - test -f ${PREFIX}/{{ install_base }}/bin/generate_cert  # [unix]
+        - if not exist %PREFIX%\bin\generate_cert exit 1  # [win]
+
+  - name: heavydb
+    version: {{ version }}
+    script: build-heavydb.sh   # [linux64]
+    script: build-heavydb.bat  # [win]
+    build:
+      string: {{ build_string_ext }}
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0")]
+      skip: true  # [osx]
+      skip: true  # [win and cuda_compiler_version != "None"]
+      missing_dso_whitelist:
+        - '*/libcuda.*'  # [cuda_compiler_version not in (undefined, "None")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      ignore_run_exports:
+        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
+        - fmt
+        - gflags
+        - glog
+        - libclang-cpp
+        - libkml
+        - ncurses
+        - openldap
+        - zlib
+      run_exports:
+        - {{ pin_subpackage('heavydb',  max_pin='x.x.x') }}
```

Comment:
Same question as in the other output. Explain how this works and why this is needed, please.